lib/fizzy/execute.cpp

```diff
@@ -357,6 +357,20 @@ inline void convert(OperandStack& stack) noexcept
     stack.top() = static_cast<DstT>(stack.top().as<SrcT>());
 }
 
+/// Performs a bit_cast from SrcT type to DstT type.
+///
+/// This is a formality and should be optimized to empty function in assembly.
```

Comment:
Nit: I wouldn't call it a "formality". Compiler can optimize it away, but from C++ point of view it's required to avoid UB, right?

Reply:
That's what I call formality.